pyvista/plotting/render_window_interactor.py

```diff
@@ -547,6 +547,109 @@ def enable_trackball_style(self):
+        right button dollys, the middle button spins, and the wheel
+        dollys.
+        ctrl + left button spins, shift + left button zooms,
+        ctrl + middle button pans, shift + middle button dollys,
+        ctrl + right button rotates in 3D, and shift + right button
+        dollys.
+
+        Shift + right currently varies from ParaView and may be changed
+        in the future.
+
+        To get a view most like ParaView 2D style, also use
+        :func:`pyvista.Plotter.enable_parallel_projection`.
+
+        Examples
+        --------
+        Create a simple scene with a plotter that has the
+        ParaView-like 2D style:
+
+        >>> import pyvista as pv
+        >>> plotter = pv.Plotter()
+        >>> _ = plotter.add_mesh(pv.Cube(center=(1, 0, 0)))
+        >>> _ = plotter.add_mesh(pv.Cube(center=(0, 1, 0)))
+        >>> plotter.show_axes()
+        >>> plotter.enable_parallel_projection()
+        >>> plotter.enable_paraview_2D_style()
+        >>> plotter.show()  # doctest:+SKIP
+        """
+        self._style = 'TrackballCamera'
+        self._style_class = None
+        self.update_style()
+
+        def _left_button_press_callback(_obj, event):  # pragma: no cover
+            """Left button press behavior."""
+            if self.interactor.GetControlKey():
+                self._style_class.StartSpin()
+            elif self.interactor.GetShiftKey():
+                self._style_class.StartDolly()
+            else:
+                self._style_class.StartPan()
+            self._style_class.OnLeftButtonDown()
+
+        def _left_button_release_callback(_obj, event):  # pragma: no cover
+            """Turn off all left button behavior."""
+            self._style_class.EndSpin()
+            self._style_class.EndPan()
+            self._style_class.EndDolly()
+            self._style_class.OnLeftButtonUp()
+
+        def _middle_button_press_callback(_obj, event):  # pragma: no cover
+            """Middle button press behavior."""
+            if self.interactor.GetControlKey():
+                self._style_class.StartPan()
+            elif self.interactor.GetShiftKey():
+                self._style_class.StartDolly()
+            else:
+                self._style_class.StartSpin()
+            self._style_class.OnMiddleButtonDown()
+
+        def _middle_button_release_callback(_obj, event):  # pragma: no cover
+            """Turn off all middle button behavior."""
+            self._style_class.EndSpin()
+            self._style_class.EndPan()
+            self._style_class.EndDolly()
+            self._style_class.OnMiddleButtonUp()
+
+        def _right_button_press_callback(_obj, event):  # pragma: no cover
+            """Right button press behavior."""
+            if self.interactor.GetControlKey():
+                self._style_class.StartRotate()
+            elif self.interactor.GetShiftKey():
+                self._style_class.StartDolly()  # ParaView does a different type of zooming
```

Comment:
ParaView 2D style does some sort of different type of zooming that I cannot figure out. I tried some of the other `Start*` modes available but some did nothing me and the documentation on these methods is sparse in vtk.

I put in a note that this behavior could change in the future

Reply:
Let's add the PR number so we can see this comment later.
```suggestion
                self._style_class.StartDolly()  # ParaView does a different type of zooming (#5336)
```

Reply:
Will this link appropriately, should we put in a direct link instead?

Reply:
Sure. If you don't mind the length of it.
```suggestion
                self._style_class.StartDolly()  # ParaView does a different type of zooming (https://github.com/pyvista/pyvista/pull/5336#discussion_r1426060132)
```

Reply:
Or...
```suggestion
                # ParaView does a different type of zooming
                # https://github.com/pyvista/pyvista/pull/5336#discussion_r1426060132
                self._style_class.StartDolly()
```